### DISCUSSION TITLE
import '../generated_bridge.dart'  cannot find declaration

### DISCUSSION THREAD

**Discussion by 2013CYB** [rustdesk/rustdesk#955] [Q&A]
when I import the fie  generated_bridge.dart  `import '../generated_bridge.dart'`, but android studio throw out the exception ’cannot find declaration‘ . I  found two issues #620 and #708   I don't know the file  ’ generated_bridge.dart’ how to  generate 
I need some help,thanks.

**Comment by cvabm** [comment]
hi, do you reloved？

**Reply by 2013CYB** [reply]
I did it. the file 'generated_bridge.dart' is generated when run the project. As to why this problem is thrown, I found that I haven't import all necessary files, I delete local project and clone several times and import files ,then  it work
